### DISCUSSION TITLE
Export image changes the original background image

### DISCUSSION THREAD

**Discussion by RitaBeloMoreira** [vinothpandian/react-sketch-canvas#92] [Q&A] [answered]
**Describe the bug**
Hi,

When I export the image with background (exportWithBackgroundImage), the final image is cropped:

-- Original Image

<img width="708" alt="Screenshot 2022-09-29 at 15 21 30" src="https://user-images.githubusercontent.com/103126811/193058422-a57ddde4-f157-47bc-92a8-ace56fa5904e.png">

-- Final Image with preserveBackgroundImageAspectRatio="none"

<img width="421" alt="Screenshot 2022-09-29 at 15 21 48" src="https://user-images.githubusercontent.com/103126811/193058622-8ad9fa8f-82e0-476b-bb4e-a30c7a0693d7.png">

-- Final Image with preserveBackgroundImageAspectRatio="xMidYMid meet"

<img width="409" alt="Screenshot 2022-09-29 at 15 21 35" src="https://user-images.githubusercontent.com/103126811/193058749-a311c8b3-16e3-4866-9739-876e8c9161ca.png">

Can you please help me?
Thanks!

Tested here: https://vinoth.info/react-sketch-canvas/

**Comment by danielvdm2000** [comment, marked as answer]
I just ran into the same problem.

The problem arises because the background image is drawn on the canvas in the original size.

So, if the image is bigger than the canvas it will be cropped.

My solution was to resize the image to the same size as the canvas before passing it to the backgroundImage prop.

**Reply by belugawhale3** [reply]
How did you get the size of the canvas?

**Comment by RitaBeloMoreira** [comment]
> I just ran into the same problem.
> 
> The problem arises because the background image is drawn on the canvas in the original size.
> 
> So, if the image is bigger than the canvas it will be cropped.
> 
> My solution was to resize the image to the same size as the canvas before passing it to the backgroundImage prop.

Thanks for your answer, that works!!
